egs/babel_multilang/s5/local/nnet3/run_tdnn_multilingual.sh

```diff
@@ -204,9 +204,12 @@ EOF
     echo " output-layer name=output-${lang_index} dim=$num_targets max-change=1.5"
   done >> $dir/configs/network.xconfig
 
+  cat <<EOF > $dir/configs/edits.config
+  rename-node old-name=output-0 new-name=output
+EOF
   steps/nnet3/xconfig_to_configs.py --xconfig-file $dir/configs/network.xconfig \
     --config-dir $dir/configs/ \
-    --nnet-edits="rename-node old-name=output-0 new-name=output"
+    --nnet-edits=$dir/configs/edits.config
```

Comment:
I modified the script to accept nnet-edits as filename and use it using edits-config. If you agree with that, I will update the description and option name as edits-config.
Also edits.config passed explicitly to xconfig_to_configs.py but train.py automatically picks up edits.config at prepare_acoustic_model function.